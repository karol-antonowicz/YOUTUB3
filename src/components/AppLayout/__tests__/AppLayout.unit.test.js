import React from 'react';
import {shallow} from 'enzyme';
import {AppLayout} from '../AppLayout';

describe('AppLayout', ()=> {
    test('renders AppLayout with no props', ()=> {
        const wrapper = shallow(<AppLayout/>);
        expect(wrapper).toMatchSnapshot();
    })
    test('renders AppLayout with 1 child el', ()=> {
        const wrapper = shallow(
            <AppLayout>
                <div>Child1</div>
            </AppLayout>
        );
        expect(wrapper).toMatchSnapshot();
    })
    test('renders AppLayout with children elements', ()=> {
        const wrapper = shallow(
            <AppLayout>
                <div>Child1</div>
                <div>
                    <div>
                        <h5>Blaalalaalbla
                            <span>chuj</span>
                            dupa
                        </h5>
                    </div>
                </div>
            </AppLayout>
        );
        expect(wrapper).toMatchSnapshot();
    })
})
