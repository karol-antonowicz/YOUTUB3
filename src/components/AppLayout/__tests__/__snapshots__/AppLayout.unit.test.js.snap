// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLayout renders AppLayout with 1 child el 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
  <div>
    Child1
  </div>
</div>
`;

exports[`AppLayout renders AppLayout with children elements 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
  <div>
    Child1
  </div>
  <div>
    <div>
      <h5>
        Blaalalaalbla
        <span>
          chuj
        </span>
        dupa
      </h5>
    </div>
  </div>
</div>
`;

exports[`AppLayout renders AppLayout with no props 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
</div>
`;
